e2e/tests-dfx/identity_command.bash

```diff
@@ -389,6 +389,19 @@ default'
     assert_eq '2vxsx-fae'
 }
 
+@test "identity use: is overridden by env var DFX_IDENTITY" {
+    assert_command dfx identity new dan
+    assert_command dfx identity new frank
+    assert_command dfx identity use dan
+    assert_command dfx identity whoami
+    assert_eq 'dan'
+    DFX_IDENTITY=frank
+    export DFX_IDENTITY
+    assert_command dfx identity whoami
+    assert_eq 'frank'
+}
```

Comment:
Let's show that the `--identity` parameter overrides the env var as expected, something like this:
```suggestion
    assert_command dfx identity whoami --identity dan
    assert_eq 'dan'
}
```
though it might be nicer to add a third identity, since dan is also selected by config

Reply:
Good idea.  Doing... Dan.